spec/cancan/controller_resource_spec.rb.orig

```diff
@@ -1,708 +0,0 @@
-require 'spec_helper'
```

Comment:
how did this end up there? 😅

Reply:
I found that on  https://github.com/CanCanCommunity/cancancan/pull/452/commits/1e6b5705d1a57b5e49faa1a2732291802ba72be9 😃